app/Controller/CoGroupsController.php

```diff
@@ -74,7 +74,7 @@ class CoGroupsController extends StandardController {
    */
 
   function beforeFilter() {
-    $this->redirectTab = 'group';
+    $this->redirectTab = 'group'; // XXX legacy? remove?
```

Comment:
Yes this is legacy and can be removed (and therefore the whole `beforeFilter()` override).

Reply:
Excellent - will do :-)

Reply:
Update committed as of ee3fa84